docs/the_black_code_style/current_style.md

```diff
@@ -292,8 +292,8 @@ multiple lines. This is so that _Black_ is compliant with the recent changes in
 style guide, which emphasizes that this approach improves readability.
 
 Almost all operators will be surrounded by single spaces, the only exceptions are unary
-operators (`+`, `-`, and `~`), and power operators when both its operands are considered
-"simple". For power ops, an operand is considered "simple" if it's only a NAME, numeric
+operators (`+`, `-`, and `~`), and power operators when both operands are simple.
+For power ops, an operand is considered simple if it's only a NAME, numeric
 CONSTANT, or attribute access (chained attribute access is allowed), with or without a
 preceding unary operator.
```

Comment:
Reading it through once more, we could also say that the surrounding spaces are removed for unaries and simple powers. Now it just says that they are an exception to the rule.

Reply:
I'm a little confused, could you elaborate?

Reply:
Oh sorry, I missed this: I meant to say that we don't actually say what happens to unaries and powers. We only say that they are an exception to the rule. Maybe something like: "[the only exceptions are unaries and simple pows], for which the spaces are removed."